cg/models/demultiplex/run_parameters.py

```diff
@@ -299,3 +311,13 @@ def get_read_1_cycles(self) -> int:
     def get_read_2_cycles(self) -> int:
         """Return the number of cycles in the second read."""
         return self._read_parser.get(RunParametersXMLNodes.READ_2_NOVASEQ_X)
+
+    def get_flow_cell_model(self) -> str:
+        """Return flow cell model."""
+        consumable_infos = self.tree.findall(".//ConsumableInfo")
+        for consumable_info in consumable_infos:
+            type_element = consumable_info.find("Type")
+            if type_element is not None and type_element.text == "FlowCell":
+                name_element = consumable_info.find("Name") or consumable_info.find("Mode")
+                if name_element is not None:
+                    return name_element.text
```

Comment:
This looks sketchy 😅 Is there not an easier way since we already know we're dealing with a NovaSeqX flow cell?

Reply:
It is hard. The XML has one element per consumable, and some of them have also nodes called `Name` and `Mode`. The best way we could find was looking for the node called `FlowCell` and extracting the mode/name from it

Reply:
Yes, right now I don't see a better way since this is the only file that holds this information... perhaps grabbing it from lims? we can reconsider at a later point I suppose. keeping it for now.

Reply:
But maybe you could create a function to extract the flow cell node outside this function